modules/networking/vpn/variables.tf

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+variable "project_id" {
+  type        = string
+  description = "The ID of the project where this VPC will be created"
+}
+
+variable "network" {
+  type        = string
+  description = "The name of VPC being created"
+}
+
+variable "router_region" {
+  type        = string
+  description = "The region in which you want to create the router"
+}
+
+variable "vpn_region" {
+  type        = string
+  description = "The region in which you want to create the VPN gateway"
+}
+variable "router_name" {
+  type        = string
+  description = "The name for the router"
+  default     = "router-vm-migrate"
+}
+variable "gateway_name" {
```

Comment:
Please put an empty line between all blocks.

Reply:
Added. I will try to get the linter working to catch these.